tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -30,4 +29,6 @@ def test_add_capacity(self):
             selector='app=rook-ceph-osd',
             resource_count=result * 3
         )
-        self.ceph_cluster.cluster_health_check(timeout=1200)
+        assert ceph_health_check(
```

Comment:
no need assert imo.
This call will raise 'CephHealthException' in case not healthy and return True when does healthy so no chance to get False for the assert